internal/schemautil/flink.go

```diff
@@ -0,0 +1,27 @@
+package schemautil
+
+import (
+	"github.com/aiven/aiven-go-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func FlinkReadUpsertKafkaFromSchema(d ResourceStateOrResourceDiff) *aiven.FlinkTableUpsertKafka {
```

Comment:
Why do we need to move `FlinkReadUpsertKafkaFromSchema` from the `flink` package since we only use it there?

Reply:
whoops, sorry, my mistake, I didn't intend to